R/utilities-plotting.R

```diff
@@ -206,11 +208,11 @@
     ospDimensions$`Concentration (molar)`
   )
 
-  if (xDimensionString %in% concDimensions) {
+  if (any(xDimensionString %in% concDimensions)) {
```

Comment:
why `any`? You are assigning `yDimensionString <- unique(data$yDimension)[[1]]`, so it will always be only one string, right?

Reply:
This is correct.

`%in%` is vectorized, which means `xDimensionString %in% concDimensions` emits a vector of length 2 of logical type because `concDimensions` is of length 2.

But `if` condition needs to be a scalar. Therefore, `any()` is necessary.

Reply:
Ah ok.